pagopa/io-app
# chore: [IA-545] Remove legacy code of messages without pagination

## Short description
When we added messaging with pagination, we didn't remove the old implementation, controlling it with a local feature flag. After several months of testing it in production, we can safely remove the old implementation of messages that doesn't use pagination. This PR removes all that legacy code.

`MessagesStatus` is legacy of the old implementation, but it's still used to perform a migration from the old feature to the new one. In the future, when every user has migrated to the paginated messages, this can be safely removed. 

## List of changes proposed in this pull request
- Removed the feature flag
- Removed `DEPRECATED_` actions
- Removed unused sections of the persisted store and related reducers
- Removed old sagas
- Removed old routes
- Removed old screens

## How to test
Run the app and verify that nothing has changed (everything should work as usual).
